diff --git a/RF24Network_config.h b/RF24Network_config.h
index 9590332..3faf0e9 100644
--- a/RF24Network_config.h
+++ b/RF24Network_config.h
@@ -52,13 +52,13 @@
      * 
      * @note The MAX_PAYLOAD_SIZE is (MAIN_BUFFER_SIZE - 10), and the result must be divisible by 24.
      */
-    #define MAIN_BUFFER_SIZE 144 + 10
+    #define MAIN_BUFFER_SIZE 1514 + 10
 
     /** Maximum size of fragmented network frames and fragmentation cache. This MUST BE divisible by 24.
     * @note: Must be a multiple of 24.
     * @note: If used with RF24Ethernet, this value is used to set the buffer sizes.
     */
-    #define MAX_PAYLOAD_SIZE  MAIN_BUFFER_SIZE-10
+    #define MAX_PAYLOAD_SIZE 1514
 
     /** Disable user payloads. Saves memory when used with RF24Ethernet or software that uses external data.*/
     //#define DISABLE_USER_PAYLOADS 
@@ -181,4 +181,4 @@
 
     #endif
   #endif  //cplusplus
-#endif //RF24_CONFIG_H
\ No newline at end of file
+#endif //RF24_CONFIG_H
